### DISCUSSION TITLE
New version, copy url to clipboard does not work

### DISCUSSION THREAD

**Discussion by mattitutti** [aclap-dev/video-downloadhelper#100] [Help and Q&A] [answered]
### Your question

Any ideas how to fix this? Just received update AGAIN and now it's completely broken. Copy url to clipboard does not work anymore on any site. 

Any way to get the older version back and disable auto update?  

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.0.23 (stable)
Browser locale: fi
Production build
Built on 2024-02-14
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

 Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0

**Comment by paulrouget** [comment, marked as answer]
Yeah I did that… sorry. Tried to fix another bug, without realizing I broke the copy action.
8.2.0.24 will fix that (should come out soon).

**Reply by mattitutti** [reply]
Thank you for your answer.

Hopefully it'll work soon!
